code/solr/tasks/Solr_Reindex.php

```diff
@@ -65,7 +70,7 @@ protected function doReindex($request)
         // this is for when index names do not match the class name (this can be done by overloading getIndexName() on
         // indexes
         if ($index && !ClassInfo::exists($index)) {
-            foreach(ClassInfo::subclassesFor('SolrIndex') as $solrIndexClass) {
+            foreach(ClassInfo::subclassesFor('SilverStripe\FullTextSearch\Solr\SolrIndex') as $solrIndexClass) {
```

Comment:
It was something silly: SolrIndex wasn't being found because I didn't declare it with `use` at the top of the FullTextSearch class. It wasn't throwing any errors, which is why I didn't pick it up!